docs/release-notes/3347-job-queue-write.txt

```diff
@@ -0,0 +1,11 @@
+:orphan:
+
+**New Features**
+
+-  Add support for dynamic job modification in the job queue. Users can use the WebUI or CLI to
+   change the priority, resource pool, and queue position of jobs without having to cancel and
+   resubmit them. To update jobs with the WebUI, go to the **Job Queue** Section and find the
```

Comment:
nit: 
- `Section` lowercase
- with the webui: in the webui or through the webui?

Reply:
good catch - also probably `through` as opposted to `in`